packages/radio-button-react/documentation/Example.tsx

```diff
@@ -20,13 +21,17 @@ const Example = () => {
             setVariant(undefined);
         }
     };
+    const style = inverted ? { backgroundColor: "#000", color: "#FFF" } : undefined;
 
     return (
-        <section className="jkl-spacing--top-3 jkl-spacing--bottom-3">
+        <section className="jkl-spacing--top-3 jkl-spacing--bottom-3" style={style}>
             <div className="jkl-spacing--bottom-3">
                 <button className="jkl-spacing--right-1" onClick={() => setInline(!inline)}>
                     Toggle inlined radio buttons
                 </button>
+                <button className="jkl-spacing--right-1" onClick={() => setInverted(!inverted)}>
+                    Toggle inverted
```

Comment:
På button så heter denne bryteren "Iverterte farger".
I Select/Textinput så heter disse bryterne dark mode. 

Vi bør vel kalle det samme for det samme?

Reply:
Enig! Jeg tenker at `inverted` gir mening på komponentnivå, og så kan man legge til rette for en dark mode på samme måte som compact mode:

```scss
.jkl-component {
    // vanlige stiler
    &--inverted,
    *[data-darkmode] & {
        // inverterte stiler
    }
}
```